doc/user/command-line.xml

```diff
@@ -943,8 +943,7 @@ prog.c
       a program for release,
       and that the value of this variable
       should be added to the command line
-      with the appropriate <literal>-D</literal> option
-      (or other command line option)
+      with the appropriate define
```

Comment:
"appropriate preprocessor argument defined" ?

Reply:
The idea was to move to a more agnostic wording, since MSVC doesn't use `-D` and I don't like the "or other command line option" bit; if you prefer the addition of the word preprocessor, I'm willing. Or "macro replacement option"?